troposphere/static/js/components/requests/ResourceHistoryMaster.jsx

```diff
@@ -122,3 +107,5 @@ export default React.createClass({
         );
     }
 });
+
+export default subscribe(MyResourceRequestsPage, ["ProfileStore", "StatusStore", "ResourceRequestStore"]);
```

Comment:
Yeah, basically. I know it's not necessary, just a nice interface to keep in mind. Could also be called 
```
subscribe([store1, store2])(myComponent)
```